projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts

```diff
@@ -37,13 +43,16 @@ export class AngularPlugin extends BaseTelemetryPlugin {
   initialize(config: IConfiguration & IConfig, core: IAppInsightsCore, extensions: IPlugin[], pluginChain?: ITelemetryPluginChain) {
     super.initialize(config, core, extensions, pluginChain);
     const ctx = this._getTelCtx();
-    const extConfig = ctx.getExtCfg<IAngularExtensionConfig>(this.identifier, { router: null });
-    CoreUtils.arrForEach(extensions, ext => {
+    const extConfig = ctx.getExtCfg<IAngularExtensionConfig>(this.identifier, { router: null, errorServices: null});
+    arrForEach(extensions, ext => {
         const identifier = (ext as ITelemetryPlugin).identifier;
         if (identifier === 'ApplicationInsightsAnalytics') {
             this.analyticsPlugin = (ext as any) as IAppInsights;
             if (ApplicationinsightsAngularpluginErrorService.instance !== null) {
               ApplicationinsightsAngularpluginErrorService.instance.plugin = this.analyticsPlugin;
+              if (extConfig.errorServices) {
```

Comment:
Should also check that it is an array -- while TypeScript blocks compiling and passing non-matching values you can still pass anything via pure JS or just by casting to any.